shared/login/forms/intro.render.desktop.js

```diff
@@ -3,46 +3,57 @@
 
 import React, {Component} from 'react'
 import {globalColors, globalStyles} from '../../styles/style-guide'
-import {Text, Icon, Button} from '../../common-adapters'
+import {Text, Icon, Button, Box} from '../../common-adapters'
 
 import type {IntroProps} from './intro.render'
 
-export default class Intro extends Component {
-  props: IntroProps;
-
+export default class Intro extends Component<void, IntroProps, void> {
   render () {
     return (
-      <div style={styles.loginForm}>
+      <Box style={{...stylesLoginForm, marginTop: this.props.justRevokedSelf ? 0 : 45}}>
+        {this.props.justRevokedSelf && <Box style={stylesRevoked}>
```

Comment:
You forgot to add this to the flow type file. The reason it didn't complain is because you explicitly check for it before using it so it won't ever really error.